README.md

```diff
@@ -668,6 +668,20 @@ npm run build -- --with-netlify
 netlify deploy
 ```
 
+### AWS Lambda (experimental)
+
+```
+npm run build -- --with-aws-lambda
+```
+
+The generated output format is compatible with [AWS Lambda](https://aws.amazon.com/lambda/).
+The output entrypoint `dist/serve.js` is compatible with [AWS Lambda format](https://docs.aws.amazon.com/lex/latest/dg/lambda-input-response-format.html).
+
+You can zip and upload the `dist` folder [manual to AWS console](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-package.html) or use one of the tools to deploy:
```

Comment:
I don't try it, but just to confirm, it works as described, right?

Reply:
yes - I do not manual upload, I use the [serverless framework](https://www.serverless.com) - you can find my demo here https://github.com/aheissenberger/waku-aws-lambda-demo

Reply:
Here is a LIVE Demo
https://d1pvniolnanw17.cloudfront.net
(will be removed soon)